packages/react-ux-capture-example/src/Lazy.jsx

```diff
@@ -17,13 +16,8 @@ export default class Lazy extends React.Component {
 	}
 	render() {
 		if (!this.state.loaded) {
-			return null;
+			return <React.Fragment>{this.props.fallback}</React.Fragment> || null;
```

Comment:
I'm curious why this needs a `<React.Fragment>` wrapper? I think `this.props.fallback` should work, even without `|| null`?